scheduler-server/sql/scheduler/tables/scheduling_condition.sql

```diff
@@ -0,0 +1,54 @@
+create table scheduling_condition (
+  id integer generated always as identity,
+  revision integer not null default 0,
```

Comment:
:large_blue_circle: I've looked through the code base (albeit quickly) and don't see that we use the revision field here, or on the scheduling goals. Is it used somewhere I'm not seeing (likely)? Or perhaps is it being included with an eye to a future capability that I'm not seeing?

Reply:
You're right - we don't currently use the `revision` field on scheduling goals or conditions - but it is auto-updated via trigger, and it could be useful (in the future) to know that a particular scheduling run used an older version of one of these inputs. Additionally, users can use these values via the API to track these revisions themselves, if they like. So - I'm going to go with:

> it being included with an eye to a future capability

even though if I'm being honest, I didn't think about it until you asked.